### PR TITLE
cf keycrypt attempt #2+

### PR DESCRIPTION
**Pull request by vpodzime**
Let's merge it this time!

Merge together:
https://github.com/cfengine/core/pull/4051
https://github.com/cfengine/enterprise/pull/590
https://github.com/cfengine/buildscripts/pull/678

### CODE REVIEW

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4582/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4582/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4582/

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4590/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4590/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4590/

**Comment by vpodzime** [comment]
The tests need to be adapted to the new output format.

**Comment by vpodzime** [comment]
@olehermanse for a review, I think it's easier to go through https://github.com/cfengine/core/pull/4051/files than checking individual commits. But I'd like to keep them all unsquashed for log and history.

**Comment by vpodzime** [comment]
@cf-bottom jenkins, please

**Comment by cf-bottom** [comment]
Sure, I triggered a build:

[Build Status](https://ci.cfengine.com//job/pr-pipeline/4591/)

**Jenkins:** https://ci.cfengine.com/job/pr-pipeline/4591/

**Packages:** http://buildcache.cfengine.com/packages/testing-pr/jenkins-pr-pipeline-4591/

File: tests/acceptance/testall [hunk 1]
```diff
@@ -99,6 +99,7 @@ CF_PROMISES=${CF_PROMISES:-}        ; export CF_PROMISES
 CF_SERVERD=${CF_SERVERD:-}          ; export CF_SERVERD
 CF_EXECD=${CF_EXECD:-}              ; export CF_EXECD
 CF_KEY=${CF_KEY:-}                  ; export CF_KEY
+CF_KEY=${CF_KEYCRYPT:-}             ; export CF_KEYCRYPT
```
Comment (olehermanse) [inline tests/acceptance/testall]: `CF_KEYCRYPT=`

**Comment by nickanderson** [comment]
```
ヽ(´ー｀)ノ
```

**Pull request merged**